Makefile

```diff
@@ -103,6 +103,10 @@ test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test -mod=mod ./controllers/... -coverprofile cover.out
 
 
```

Comment:
You will need the ENVTEST variable defined before using it here like this https://github.com/openshift/oadp-operator/blob/ef445b09b303262d27f59c84a78e4c0a9e01bcb6/Makefile#L144

Reply:
if you don't, the test-ci command might fail eventually